.github/workflows/ci.yml

```diff
@@ -0,0 +1,110 @@
+name: ci
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - '**'
+env:
+  COVERAGE_OPTION: ./node_modules/.bin/nyc
```

Comment:
@davimacedo just wondering, what is `COVERAGE_OPTION` used for? I haven't found any references.

Reply:
I believe it makes [this](https://github.com/parse-community/parse-server/pull/7321#issuecomment-813426432) to work.

Reply:
How so?

I believe it was part of a script in package.json:

`cross-env NODE_ENV=test TESTING=1 ./node_modules/.bin/babel-node $COVERAGE_OPTION ./node_modules/jasmine/bin/jasmine.js`

Reply:
You might be right. Let's try to remove it and see what happens.